docs/source/releasenotes.rst

```diff
@@ -20,15 +27,12 @@ Release notes
 
   - Improved locators parsing and ability to enclose values containing spaces with
     ``"`` double-quote. (:issue:`363`)
+  -
 
   .. warning::
     This is a **breaking** change! If you use single-quote locator value enclosing,
```

Comment:
This breaking change is brought outside of this release as well on fresh built envs. (because `windows` automatically updates its `core` to the latest "non-breaking")